'initial' contribution (#13)
swcarpentry/python-intermediate: _episodes/04-datastructures.md

---
title: Data Structures in Python
teaching: 50
exercises: 15
questions:

> - "How do I use more complex data structures"
> - "Why is it useful?"

Reviewer: ```suggestion
- "What are complex data structures?"
- "How and when to use them?"
```
This is still a bit fuzzy. While developing this episode, clarify which data structures you intend to cover (probably `list`, `dict`, `tuple`? )
